src/nccl_ofi_net.c

```diff
@@ -2251,10 +2262,12 @@ static ncclResult_t ofi_closeRecv(void *recvComm)
 		rc = fi_close((fid_t)mr_handle);
 		if (OFI_UNLIKELY(rc != 0)) {
 			ret = ncclSystemError;
-			NCCL_OFI_WARN("Unable to de-register memory. RC: %d, Error: %s",
+			NCCL_OFI_WARN("Unable to de-register flush buffer. RC: %d, Error: %s",
 				      fi_strerror(-rc));
 			goto exit;
 		}
+		munmap(rComm->flush_buff.host_buffer, sysconf(_SC_PAGESIZE));
```

Comment:
return value checks for munmap?

Reply:
Thought about it.  Check the return value and then do what?  Seems like proceeding with the teardown is the right approach.  I guess this could log a message.